pym/bob/input.py

```diff
@@ -3465,7 +3465,7 @@ def __parse(self, envOverrides, platform):
             env.update(self.__defaultEnv)
         env.setFuns(self.__stringFunctions)
         env.update(envOverrides)
-        env["BOB_HOST_PLATFORM"] = platform
+        env["BOB_HOST_PLATFORM"] = platform if not isMsys else 'msys'
```

Comment:
This is the wrong spot. In case of Jenkins the `platform` variable describes the Jenkins architecture which might be different from the one where Bob is running. Actually the

```python
    def parse(self, envOverrides={}, platform=sys.platform):
```

above must be changed to something like

```python
    def parse(self, envOverrides={}, platform=getPlatformString()):
```

where `getPlatformString()` needs to be defined in utils.py.